js/packages/teams-ai/src/prompts/PromptSection.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @module teams-ai
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { TurnContext } from 'botbuilder';
+import { Tokenizer } from '../tokenizers';
+import { PromptFunctions } from './PromptFunctions';
+import { Message } from './Message';
+import { Memory } from '../MemoryFork';
+
+export interface PromptSection {
```

Comment:
Leave some docs that this is for token budgeting? Brief benefits of having sections?